### PR TITLE
Add link to the TypeScript cookiecutter in the Low Level docs

### PR DESCRIPTION
**Pull request by jtpio**

### CODE REVIEW

**Comment by pbugnion** [comment]
Thanks for submitting this.

I worry that the current phrasing makes things complicated for developers unfamiliar with the complexities of JavaScript module implementations (what's CommonJS?).

Basically, we're saying:
- if you don't want to deal with the complexities of Webpack, typescript and the JavaScript toolchain, choose CommonJS.
- if you're somewhat familiar with JavaScript tooling, use the typescript cookiecutter.

Maybe solving this is as simple as adding a description next to the CommonJS link saying: "Use this if you are less familiar with frontend build tools."?

**Comment by jtpio** [comment]
The JS cookiecutter still deals with Webpack and the usual JavaScript toolchain.

The main reason to have it say "CommonJS" was to have something consistent with the JupyterLab docs: https://jupyterlab.readthedocs.io/en/stable/developer/extension_dev.html#plugins

The main difference between these two cookiecutters is that one is using JavaScript, and the other TypeScript. It would also be fine to say "JavaScript" instead of "CommonJS".

**Comment by pbugnion** [comment]
> The JS cookiecutter still deals with Webpack and the usual JavaScript toolchain.

You're right -- I was thinking of an older version that didn't do this, I think. Or maybe just confused.

I think changing CommonJS -> JavaScript makes it easier for people to understand what they're getting into.

File: docs/source/examples/Widget Low Level.ipynb [hunk 1]
```diff
@@ -516,11 +516,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "A template project is available in the form of a cookie cutter: https://github.com/jupyter-widgets/widget-cookiecutter\n",
+    "Two template projects is available in the form of cookiecutters\n",
```
Comment (pbugnion) [inline docs/source/examples/Widget Low Level.ipynb]: is -> are

File: docs/source/examples/Widget Low Level.ipynb [hunk 2]
```diff
@@ -516,11 +516,14 @@
...
     "\n",
-    "This project is meant to help custom widget authors get started with the packaging and the distribution of Jupyter interactive widgets.\n",
+    "- CommonJS: https://github.com/jupyter-widgets/widget-cookiecutter\n",
+    "- TypeScript: https://github.com/jupyter-widgets/widget-ts-cookiecutter\n",
     "\n",
-    "It produces a project for a Jupyter interactive widget library following the current best practices for using interactive widgets. An implementation for a placeholder \"Hello World\" widget is provided."
+    "These cookiecutters are meant to help custom widget authors get started with the packaging and the distribution of Jupyter interactive widgets.\n",
+    "\n",
+    "They produce a project for a Jupyter interactive widget library following the current best practices for using interactive widgets. An implementation for a placeholder \"Hello World\" widget is provided."
```
Comment (pbugnion) [inline docs/source/examples/Widget Low Level.ipynb]: Maybe "They produce a project for a Jupyter interactive widget library" -> "They produce a project for a widget library"

i.e. drop "Jupyter interactive". By now, people should know what we're talking about and the sentence feels a little stilted.

**Comment by jtpio** [comment]
Thanks @pbugnion for the suggestions!

**Comment by pbugnion** [comment]
Looks great, thank you!

**Pull request merged**